Xwt/Xwt.Drawing/Font.cs

```diff
@@ -131,13 +131,18 @@ internal static Font FromName (string name, Toolkit toolkit)
 			int i = name.LastIndexOf (' ');
 			int lasti = name.Length;
 			do {
+				if (lasti > 0 && IsFontSupported (name.Substring (0, lasti)))
+					break;
+				
 				string token = name.Substring (i + 1, lasti - i - 1);
```

Comment:
I know this probably isn't related to the PR, but what if the name is empty?

We'd end up with `name.Substring (1, -1);` on a 0 Length string. Not ok. We should probably assert a `string.IsNullOrWhitespace (name)` at the beginning.